tests/testsuite/cargo_information/path_dependency/stdout.log

```diff
@@ -0,0 +1,6 @@
+foo
+version: 0.0.0 (from ./)
+license: unknown
+rust-version: unknown
+dependencies:
+  crate1@* (./crates/crate1)
```

Comment:
Should we drop `@*`?  If its a straight wildcard, it is likely unset and is extra noise

Reply:
I dropped it for all non-registry deps. What about `bar = {version = 1.2.3, git = "xxx"}`? Should we display the version request here?

Reply:
I lean towards not but understand if you think we should.

Reply:
Thanks! I also prefer not to show it.